tests/unit/ui/components/TestRating.js

```diff
@@ -40,13 +66,29 @@ describe('ui/components/Rating', () => {
   });
 
   it('can be classified as small', () => {
-    const root = render({ styleName: 'small' });
+    const root = render({ styleSize: 'small' });
     expect(root.element.className).toContain('Rating--small');
   });
 
-  it('throws an error for invalid styleNames', () => {
-    expect(() => render({ styleName: 'x-large' }))
-      .toThrowError(/styleName=x-large is not a valid value; possible values: small,/);
+  it('is classified as owned if you wrote the review', () => {
+    const review = signInAndDispatchSavedReview({
+      siteUserId: 123, reviewUserId: 123,
+    });
+    const root = render({ review });
+    expect(root.element.className).toContain('Rating--by-owner');
+  });
+
+  it('is not classified as owned if you did not write the review', () => {
```

Comment:
This needs one more test where no user is signed into the site. You can either do it by calling [`dispatchClientMetadata()`](https://github.com/mozilla/addons-frontend/blob/master/tests/unit/amo/helpers.js#L203) (which sets up an initial logged out state) or dispatching [`logOutUser()`](https://github.com/mozilla/addons-frontend/blob/master/src/amo/reducers/users.js#L156) (sometimes that's easier).